A62-pick-first.md

```diff
@@ -9,63 +9,68 @@ A62: `pick_first` LB policy
-behavior can be summarized in a few sentences, as done above, there are some
-subtle differences in implementation across the various languages. This document
-aims to provide a more detailed description of its behavior, with the aim of
-bringing convergence among the different implementations.
+When connections to all addresses fail, there are some similarities and some
+differences between the Java/Go implementations and the C-core implementation.
 
-## Proposal
+Similarities include:
+- Reporting `TRANSIENT_FAILURE` as the connectivity state to the channel.
+- Applying exponential backoff. During the backoff period, `wait_for_ready` RPCs
+  are queued while other RPCs fail.
+
+Differences show up after the backoff period ends:
+- C-core remains in `TRANSIENT_FAILURE` while Java/Go move to `IDLE`.
+- C-core attempts to reconnect to the given addresses, while Java/Go rely on the
+  client application to make an RPC or an explicit attempt to connect.
+- C-core moves to `READY` only when a connection attempt succeeds.
 
-Specific scenarios are described in their own subsections below.
+This behavior of staying in `TRANSIENT_FAILURE` until it can report `READY` is
+called sticky-TransientFailure, or sticky-TF.
 
-### When connections to all addresses fail
+## Proposal
+
+Specific changes are described in their own subsections below.
 
-`pick_first` LB policy attempts to connect to the given addresses in order, and
-when none of the addresses are reachable, it must:
-- report `TransientFailure` as the connectivity state to the channel
-- apply exponential backoff (all RPCs attempted during this period will fail)
-- attempt to reconnect to the given addresses in order
-- continue to report `TransientFailure` as the connectivity state, until a
-  connection succeeds, at which point, it must report `Ready`
+### Support Sticky-TF
 
-This behavior of staying in `TransientFailure` until it can report `Ready` is
-called sticky-TransientFailure.
+All `pick_first` implementations should support sticky-TF. Once connections to
+all addresses fail, they should:
+- Report `TRANSIENT_FAILURE` as the connectivity state.
+- Attempt to reconnect to the addresses indefinitely until a connection succeeds
+  (at which point, they should report `READY`), or the channel becomes idle (see
+  [gRPC Connectivity Semantics](1) for more details about idleness).
 
-`pick_first` LB policy should indefinitely attempt to reconnect, in a bid to
-move out of `TransientFailure` and into `Ready`, until the channel enters idle
-mode due to inactivity, at which point, the channel will shut down the name
-resolver and the LB policy. See [gRPC Connectivity Semantics](1) for more
-details about idleness.
+Supporting sticky-TF has the following advantages:
+- Avoids long delays before failing RPCs while the channel goes back to
+  `CONNECTING` state while attempting to reconnect.
+- Allows `pick_first` to work as a child of the [priority LB policy](2).
```

Comment:
It might be specifically worth noting that one example of where this comes up is when a LOGICAL_DNS cluster is used under an aggregate cluster, where the LOGICAL_DNS cluster is not the last cluster in the list.  You can reference gRFC A37 here.

Reply:
Added a note here. Not sure if I made it too verbose though.